src/scss/misc.scss

```diff
@@ -253,6 +253,27 @@ form, .form {
         }
     }
 
+    .radio {
```

Comment:
Curious, was there a reason additional Radio styling was necessary vs. the built-in bootstrap form-check/form-check-input/form-check-label styling used in other components?

Reply:
@cscharf We only use bootstrap on the web vault. Other apps are completely custom styling for all elements (some conventions borred from bootstrap, however).